client/src/Components/ContactForm.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { useFormik } from 'formik';
+import * as Style from './ContactForm.styles'
+
+//Here we set up the initial settings and functions to handle and validate the forms while using Formik.
+const initialValues = {
+  name: '',
+  email: '',
+  comments: ''
+}
+
+//Here we handle setting info from input fields setting it to local storage
+const onSubmit = values => {
+  localStorage.setItem('contactMsg', JSON.stringify(values))
+}
+
+//here is Form validation, checking if fields have info in them and email validation regex
+const validate = values => {
+  let errors = {}
+
+  if(!values.name) {
+      errors.name = 'Required'
+  } else if (values.name.length < 3) {
+    errors.name = 'Must be atleast 3 characters'
+  }
+
+  if (!values.email) {
+      errors.email = 'Required';
+    } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
+      errors.email = 'Invalid email address';
+    }
+
+  if(!values.comments) {
+      errors.comments = 'Required'
+  } else if (values.comments.length < 20) {
+    errors.comments = 'Must be atleast 20 characters'
+  }
+
+  return errors
+}
+
+const ContactForm = () => {
+  const formik = useFormik({
```

Comment:
variable can be destructured

Reply:
Destructured variable, also refactored the code, to make it more readable, by using `getFieldProps` method